content/release-notes/2023-05-01-console.md

```diff
@@ -0,0 +1,14 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- Control Plane: Added support for configuring the PostgreSQL [default_text_search_config](https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-DEFAULT-TEXT-SEARCH-CONFIG) configuration parameter, which determines the behavior of text search functions and operators that do not have an explicit configuration. This parameter can be set for the PostgreSQL instance using the `default_endpoint_settings` property in the [Create project](https://api-docs.neon.tech/reference/createproject) or [Update project](https://api-docs.neon.tech/reference/updateproject) API, or using the `pg_settings` property in the [Create endpoint](https://api-docs.neon.tech/reference/createprojectendpoint) or [Update endpoint](https://api-docs.neon.tech/reference/updateprojectendpoint) API. Alternatively, the parameter can be set for a specific database using `ALTER DATABASE` syntax:
+
+  ```sql
+  ALTER DATABASE <dbname> SET default_text_search_config = ‘pg_catalog.english’;
+  ```
+
+- Integrations: Updated the [Neon Vercel Integration](../docs/guides/vercel) to require selecting a database and role when connecting a Vercel project to a Neon database.
```

Comment:
this was rather a bugfix, which was not even visible to users, rather caught in tests. Let's omit it?

Reply:
I am talking about vercel integration step

Reply:
Thank you. Dropped the Integrations item.